etc/frontend/events.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
+    <event name="customer_register_success">
+        <observer name="InchooAddDefaultBookmarkList" instance="Inchoo\ProductBookmark\Observer\AddDefaultBookmarkList" />
```

Comment:
što ako si modul instalirao na neki projekt koji već ima hrpu customera? Kako ćeš onda assignati default list? Ja bi kreiranje ovoga prebacio negdje drugdje, npr. kada netko želi nešto bookmarkati, a nema listu, kreirao bi ju.

Reply:
Maknio observer, ako user nema listu - moze bookmarkati i napravi se default lista.